lib/tasks/test_providers_common.rake

```diff
@@ -0,0 +1,15 @@
+require_relative './evm_test_helper'
+
+if defined?(RSpec)
+namespace :test do
+  desc "Run all specs tagged 'providers_common'"
+  RSpec::Core::RakeTask.new(:providers_common => ["test:initialize", "evm:compile_sti_loader"]) do |t|
+    EvmTestHelper.init_rspec_task(t, ['--tag', 'providers_common'])
+
+    if defined?(ENGINE_ROOT)
+      t.rspec_opts += ["--exclude-pattern", "manageiq/spec/tools/**/*_spec.rb"]
```

Comment:
Note that the 'tools/miq_ldap_to_sssd_spec.rb' does a require on 'tools/miq_ldap_to_sssd.rb', which it turn requires the Rails environment, which blows up from the plugin side.  Excluding all of tools for now is a workaround, but we should fix miq_ldap_to_sssd_spec to not do that.

Even so, we have to set *some* exclude pattern, even to a bogus value, otherwise the default exclude pattern kicks in which ignores *all* of the manageiq/spec directory, effectively preventing this test from running at all.